target/linux/ath79/nand/base-files/etc/board.d/02_network

```diff
@@ -20,6 +20,11 @@ ath79_setup_interfaces()
 		ucidef_add_switch "switch0" \
 			"0@eth0" "2:lan:2" "3:lan:1" "1:wan"
 		;;
+	netgear,r6100)
+		ucidef_set_interfaces_lan_wan "eth0.1" "eth1"
+		ucidef_add_switch "switch0" \
+			"0@eth0" "1:lan:1" "2:lan:2" "3:lan:3" "4:lan:4"
```

Comment:
The suffixes can be dropped (only needed if they deviate from the internal ports):
"0@eth0" "1:lan" "2:lan" "3:lan" "4:lan"